src/GasExposureAnalytics.py

```diff
@@ -223,7 +223,7 @@ def __init__(self, list_of_csv_files=None, config_filename=DEFAULT_CONFIG_FILENA
             dataframes = []
             for csv_file in list_of_csv_files : 
                 df = pd.read_csv(csv_file, engine='python', parse_dates=[TIMESTAMP_COL], index_col = TIMESTAMP_COL)
-                assert FIREFIGHTER_ID_COL in df.columns, "CSV files is missing key columns %s" % (required_cols)
+                # assert FIREFIGHTER_ID_COL in df.columns, "CSV files is missing key columns %s" % (required_cols)
```

Comment:
Checking with @JSegrave-IBM. Created https://github.com/Pyrrha-Platform/Pyrrha-Rules-Decision/issues/38 for @TaeBlack @Aquijada52